helix-loader/src/main.rs

```diff
@@ -5,5 +5,6 @@ use helix_loader::grammar::fetch_grammars;
 // compilation time. This is not meant to be run manually.
 
 fn main() -> Result<()> {
+    println!("uo");
```

Comment:
You should delete this.

Reply:
Wow. I have no idea how or why that's there. I'll go through and fix lint issues and of course remove this. Thanks!